bindings/python/src/OpenSpaceToolkitAstrodynamicsPy.cxx

```diff
@@ -45,6 +46,7 @@ PYBIND11_MODULE (OpenSpaceToolkitAstrodynamicsPy, m)
 	OpenSpaceToolkitAstrodynamicsPy_Trajectory(m) ;
 	OpenSpaceToolkitAstrodynamicsPy_Flight(m) ;
 	OpenSpaceToolkitAstrodynamicsPy_Access(m) ;
+    OpenSpaceToolkitAstrodynamicsPy_NumericalSolver(m) ;
```

Comment:
indentation

Reply:
And add comment expressing the fact that `NumericalSolver` will be moved to `ostk-mathematics` in the future maybe for clarity